routes/nodebb.js

```diff
@@ -12,6 +12,7 @@ const telemetryHelper = require('../helpers/telemetryHelper.js')
 const methodSlug = '/update';
 const nodebbServiceUrl = NODEBB_SERVICE_URL+ nodebb_api_slug;
 const _ = require('lodash')
+const triggerNotification = true;
```

Comment:
This is not required

Reply:
Instead of this, there should be env variable to trun-off notification for all API's
@VenkatKadiveti